### DISCUSSION TITLE
Integrate with LibreWolf browser possible?

### DISCUSSION THREAD

**Discussion by Ricardo-Simoes** [keepassxreboot/keepassxc#8394] [Ideas]
Is there any chance of making integration with **LibreWolf** browser possible? LibreWolf is a fork of Firefox, focused on privacy, security and freedom. https://librewolf.net/

**Comment by droidmonkey** [comment]
Use the custom browser option. We have no plans to officially support this browser.

**Reply by Ricardo-Simoes** [reply]
Already tried, but can't make it work, says "keys can not be exchanged", maybe I'm doing something wrong...

**Reply by droidmonkey** [reply]
Please read the troubleshooting guide for the browser extension

**Comment by enoironline** [comment]
hello droidmonkey, can you please pin the link to the troubleshooting guide for KeepassXC/Librewolf under Mac?

**Reply by enoironline** [reply]
maybe this link? : https://github.com/keepassxreboot/keepassxc-browser/wiki/Troubleshooting-guide
No fix for KeepassXC/Librewolf on Mac?
